extra/cloudformation/default-unauth-resources-template.yaml

```diff
@@ -31,6 +31,7 @@ Resources:
           - Action:
               - "mobiletargeting:PutEvents"
               - "mobiletargeting:UpdateEndpoint"
+              - "mobiletargeting:GetEndpoint"
```

Comment:
Can you create a task as a follow up to limit growers access to the unauth acess? 
I'm thinking we should use `aws:referer` as policy condition to our website to only requests from our webs to use unauth cognito.

Reply:
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_condition-keys.html

Reply:
Yes @vinguthh we can definitely do that via `aws:referer`, below are the links we can whitelist for web
- http://localhost:3000
- https://dev.amazonlocation.services
- https://qa.amazonlocation.services
- https://webapp.beta.geo.waypoint.aws.dev
- https://webapp.gamma.geo.waypoint.aws.dev
- https://location.aws.com

@wadhawh can you kindly update the template accordingly? And if we can also whitelist for iOS and Android.

Reply:
Thanks, can you help create a ticket to track this.

Reply:
Sure thing!